test-integration/util/GraqlTestUtil.java

```diff
@@ -115,4 +123,43 @@ public static Thing getInstance(Transaction tx, String id){
         }
         return things.iterator().next();
     }
+
+    public static void insertStatements(Session session, List<Statement> statements,
```

Comment:
given that this is now in Utils maybe let's explain what's happening and maybe a less generic name, something like `insertStatementsConcurrently` or `withLatch` or `maximiseContention`